src/libraries/IterableMap.sol

```diff
@@ -0,0 +1,104 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+contract IterableMap {
+
+    struct Item {
+        bool hasItem;
+        uint256 value;
+        uint256 offset;
+    }
+
+    address owner;
+    bool initialized;
+    uint256[] itemsArray;
+    Item[] itemsMap;
+    uint numberOfItems;
+
+    function any() {
+         if (this.initialized) {
+             require(msg.sender == this.owner || msg.sender == this);
+         }
+    }
+
+    function initialize(address owner) returns (uint) {
```

Comment:
Same comment as above with visibility modifiers, all functions should explicitly have visibility modifiers.  Any function that doesn't write to storage should be marked as `constant`, the rest should probably be `public onlyOwner` (see comment above re: modifiers).